src/derive/curve.rs

```diff
@@ -68,13 +75,24 @@ macro_rules! new_curve_impl {
             () => {
                 paste::paste! {
 
+                // The copressed size is the size of the x-coordinate (one base field element)
```

Comment:
I believe `copressed` is `compressed`. right? @davidnevadoc 
I think once you merge/rebase the `main` branch, we can fix all of these typos error. 🙂